website/docs/patterns/objects.md

```diff
@@ -49,7 +53,8 @@ conf/
 ├── config.yaml
 └── db
     ├── mysql.yaml
-    └── postgresql.yaml
+    ├── postgresql.yaml
+    └── postgresql_default.yaml
```

Comment:
I think we don't need a complete example.
It's enough to say in the main doc that this supports:
 - classes
 - class methods
 - static methods
 - (and if you test) static class methods.

Reply:
Ok, where in the main doc is this?  I can add that to this PR.

Reply:
Does not exist yet, see my comment. we can add a new section to objects.md for now.